plugins/protocolbuffers/pyi/v24.4/buf.plugin.yaml

```diff
@@ -7,3 +7,11 @@ output_languages:
   - python
 spdx_license_id: BSD-3-Clause
 license_url: https://github.com/protocolbuffers/protobuf/blob/v24.4/LICENSE
+registry:
```

Comment:
note to myself to move pb/pyi and pb/python to the v25.0 version (or whatever the latest one is when we want to ultimately merge this)

Reply:
Added to v25.0 versions in https://github.com/bufbuild/plugins/pull/899/commits/85f860a49c719ee821f18988072abc2d4d175c4b